.yarnrc.yml

```diff
@@ -0,0 +1,13 @@
+nodeLinker: node-modules
```

Comment:
I don't know what anything in this directory and the .yarn directory does. I copied these from the js-core repo.